docs/howto/how_to_use_ci_result.md

```diff
@@ -1,27 +1,27 @@
 # How to Use CI Result in Apollo
-Apollo PR can be merged depend on CI result and CLA.
+In Apollo, wether a PR can be merged depends on CI result and CLA.
 
 ## What works will CI check?
-Apollo CI will run following steps:
+Apollo CI will follow the steps below:
 
-1. Build your PR code merged master
-1. Lint your PR code include cc, h, py, BUILD, etc
+1. Merge your PR into master and build
```

Comment:
We can't say that PR is **already** merged into master.
What about 
`Checkout your PR into Apollo codebase`?